packages/base-types/src/models/project/index.ts

```diff
@@ -19,6 +19,11 @@ export interface ReportTag {
   tagID: string;
   label: string;
 }
+
+export interface StickerURL {
```

Comment:
I guess just `Sticker` is better, isn't it?

```suggestion
export interface Sticker {
```